packages/@ember/debug/lib/deprecate.ts

```diff
@@ -127,7 +127,7 @@ if (DEBUG) {
 
       if (error instanceof Error) {
         if (error.stack) {
-          if (error['arguments']) {
+          if ((error as any)['arguments']) {
```

Comment:
We can use `in` here safely, esp. since we don't actually need to *use* `error.arguments` ([playgrounnd][1]):

```suggestion
          if ('arguments' in  error) {
```

[1]: https://www.typescriptlang.org/play?ts=4.6.2#code/CYUwxgNghgTiAEEQBd4gFzwK4DsDWOA9gO44BQAlgGbwAUCFOAzslDmCITQKIwyEwAlPADeZePGp0QAOhZQweYWIkSptAOSwA5lgC2IHMiYbJONMvGqJViQF8rDu0A

Reply:
Interestingly this breaks the next line that does `error.stack`.